eng/pipelines/templates/steps/verify-autorest.yml

```diff
@@ -0,0 +1,66 @@
+parameters:
+  - name: ServiceToTest
+    type: string
+    default: ''
+  - name: SDKType
+    type: string
+    default: 'client'
+  - name: ServiceDirectory
+    type: string
+  - name: NodeVersion
+    type: string
+    default: '12.x'
+  - name: PythonVersion
+    type: string
+    default: '3.9'
```

Comment:
We have a python version default available in globals.yml. Right now the default we're packages n stuff with is `3.6`. Feel free to default to `'$(PythonVersion)'` here. Just so it's cohesive. If it makes a huge difference for you, that may also mean bumping the default that we use to 3.9.

Reply:
I don't believe that will negatively affect `build-artifacts.yml` which is really the only piece I'm super concerned with.